lib/multi.c

```diff
@@ -374,6 +374,11 @@ struct Curl_multi *Curl_multi_handle(int hashsize, /* socket hash */
   multi->max_concurrent_streams = 100;
   multi->ipv6_works = Curl_ipv6works(NULL);
 
+#ifdef USE_WINSOCK
+  multi->wsa_event = WSACreateEvent();
+  if(multi->wsa_event == WSA_INVALID_EVENT)
+    goto error;
+#else
```

Comment:
Why is this independent of ENABLE_WAKEUP?

Reply:
Because the event is used for normal socket waiting as well. This single event is used for all waiting purposes.

Reply:
@jay can I consider this comment solved then?

Reply:
yes